### PR TITLE
Fix bash condition in endtoend script

### PR DESCRIPTION
**Pull request by EngHabu**
Signed-off-by: Haytham Abuelfutuh <haytham@afutuh.com>

fixes: https://github.com/flyteorg/flyte/issues/1825

tested here: https://github.com/flyteorg/flytepropeller/pull/363

### CODE REVIEW

File: boilerplate/flyte/end2end/end2end.sh [hunk 1]
```diff
@@ -18,14 +18,14 @@ pushd ${OUT}
 if [ ! -z "$IMAGE" ]; 
 then
   kind load docker-image ${IMAGE}
-  if [ ${IMAGE_NAME} -eq "flytepropeller" ]
+  if [ "${IMAGE_NAME}" == "flytepropeller" ]
   then
-    sed -i.bak -e "s_${IMAGE_NAME}:.*_${IMAGE}_g" ${OUT}/kustomize/base/propeller/deployment.yaml
+    sed -i.bak -e "s_${IMAGE_NAME}:.*_${IMAGE}_g" "${OUT}"/kustomize/base/propeller/deployment.yaml
```
Comment (kumare3) [inline boilerplate/flyte/end2end/end2end.sh:23]: should we continue to use kustomize?

File: boilerplate/flyte/end2end/end2end.sh [hunk 2]
```diff
@@ -18,14 +18,14 @@ pushd ${OUT}
...
   fi
 
-  if [ ${IMAGE} -eq "flyteadmin" ]
+  if [ "${IMAGE}" == "flyteadmin" ]
```
Comment (kumare3) [inline boilerplate/flyte/end2end/end2end.sh:26]: should this be IMAGE_NAME

File: boilerplate/flyte/end2end/end2end.sh (hunk 1 above)
Comment (EngHabu) [reply boilerplate/flyte/end2end/end2end.sh:23]: Ideally not... might as well change it now I guess...

Comment (EngHabu) [reply boilerplate/flyte/end2end/end2end.sh:23]: I'll leave that for another PR 😬 ... I want to get this in for 18.1

**Review by wild-endeavor — APPROVED** [review]

**Pull request merged**